setup.py

```diff
@@ -38,6 +38,7 @@
         "docopt>=0.6.1,<0.7",
         "breadability>=0.1.20",
         "requests>=2.7.0",
+        "pycountry==18.2.23",
```

Comment:
Please don't pin libraries in setup.py to specific versions. I want to users choose. Use min. working version please.

Reply:
Done in b35a174